docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,113 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
+<!-- Config interpolation for the gui name didn't work for the title here. -->
+
+If you want to create a Kubernetes cluster, you can do it via {{config.extra.gui}} using Gardener.
+This tutorial shows you how to do that, and how to deploy a sample application on such cluster.
+
+## Prerequisites
+
+To access the cluster from your computer, you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine.
+
+## Creating Kubernetes cluster in {{config.extra.gui}}
+
+To get started, navigate to [{{config.extra.gui_domain}}](https://{{config.extra.gui_domain}}) and in the side panel choose Kubernetes
+-> [Managed Kubernetes](https://{{config.extra.gui_domain}}/containers/gardener).
+You will be presented with a Gardener page, in which you can create and manage your clusters. Click `Create Kubernetes cluster`.
+
+![Gardener page in {{config.extra.gui}}](assets/gardener_page.png)
+
+> In Gardener's terminology, kubernetes cluster is referred as a **shoot cluster**. You can see name "Shoot" in various places throughout
+> the panel's UI, so it is good to know what it means. To learn more, check
+> out [Gardener architecture](https://gardener.cloud/docs/gardener/concepts/architecture/).
+
+In the opened form, fill in the name of the cluster and a region to see the rest of the options. Hover over question mark icons at each form
+field to learn more about it. For this tutorial, you can leave everything at default values and click `Create` at the bottom.
+
+> In the form you can see section about Worker groups. This name refers to Kubernetes Worker Nodes.
+
+In the list of clusters, you will se your new Gardener shoot bootstrapping. The icon on the left marks the progress. Creating the Cluster
+can take up to several minutes.
+
+![Shoot cluster bootstrapping](assets/shoot_bootstraping.png)
```

Comment:
I know I am being a stickler here, but please fix the file name to `shoot_bootstrapping.png` (or `shoot_bootstrap.png`; that would work too). :)